static/js/services/contact-schema.js

```diff
@@ -176,6 +176,16 @@ angular.module('inboxServices').service('ContactSchema',
         });
         return schema;
       },
+      getChildPlaceType: function(type) {
+        var schema = getSchema();
+        if (!_.has(schema, type)) {
+          return undefined;
+        }
+        var child = _.findKey(schema, function(value) {
```

Comment:
actually you don't even need that, because findKey returns undefined if it doesn't match, so this function could probably just be `return _.findKey(schema, function(value) { .. });`